connectors/katalog/pkg/connector/connector.go

```diff
@@ -57,3 +71,100 @@ func (r *Handler) getAssetInfo(c *gin.Context) {
 
 	c.JSON(http.StatusOK, &response)
 }
+
+func (r *Handler) reportError(errorMessage string, c *gin.Context, httpCode int) {
+	r.log.Error().Msg(errorMessage)
+	c.JSON(httpCode, gin.H{"error": errorMessage})
+}
+
+// Enables writing of assets to katalog. The different flows supported are:
+// (a) When DestinationAssetID is specified:
+//     Then a destination asset id is created with name : <DestinationAssetID>
+// (b) When DestinationAssetID is not specified but ResourceMetadata.Name of source asset is specified:
+//     Then an asset is created with name: ResourceMetadata.Name-<RANDOMSTRING_LENGTH_4>
+// (c) When DestinationAssetID and ResourceMetadata.Name of source asset are not specified:
+//     Then an asset is created with name: fybrik-asset-<RANDOMSTRING_LENGTH_4>
```

Comment:
`fybrik-asset` is a contstant and it can be changed. Please define a constant e.g. `FYBRIK_ASSET_PREFIX` 
I'm not sure that we need the asset substring in the prefix, we are talking about assets. 
I'd suggest `FYBRIK_ASSET_PREFIX = "fybrik-"`

Reply:
Fixed. Please check. Thanks.